TombEngine/Game/items.h

```diff
@@ -61,7 +61,11 @@ enum class EffectType
 	None,
 	Fire,
 	Sparks,
-	Smoke
+	Smoke,
+	MagicFire,
+	ColoredFire,
+	ElectricDeath,
+	LaserDeath
```

Comment:
Are these effect types specifically tied to deaths, or can they be used on their own?